servos: 
  base: [0, "base", -45, 45, 1002, 1907,-90, 90, 0.22,true]
  shoulder: [1, "shoulder", 55, 145, 1130, 1900, 27, 145, 0.19,false]
  elbow: [2, "elbow", -125, -53, 1106, 1790 , -125, -53, 0.28,true]
  wrist: [3, "wrist", -90, 90, 500, 2388, -90, 90, 0.24,false]
  wrist_rotate: [4, "wrist_rotate", -90, 90, 700, 2400, -90, 90, 0.16,false]
  gripper: [5, "gripper", -90, 90, 700, 2400, -90, 90, 0.16,false]
sim_servos:
  gripper: [2, -0.02, 0.02] 
arm_lengths: [0.0, 0.148, 0.205]